darksidemanas.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "darksidemanas"
+project: "medical sos emergeny recordig and monitoring system"
+---
+
+# Project Name
+MEDICAL SOS EMERGENCY MONITORING AND RECORDING SYSTEM 
+## Summary
+
+I WANT TO MAKE IOT DEVICE TO CONNECT PATIENTS AND ALL MEDICAL ,SO THEY CAN MONITOR MULTIPLE PEOPLE AT THE SAME TIME IT WILL HELP IN SITUATIONS LIKE COVID-19.
+I WILL BE DEVEOPING ANDRIOD APP TO DO SO.
+
+## Plan
+
+1.FIRST WE NEED TO CONFIGURE RASPBERRY PI TO READ ANALOG SIGNALS USING A CUSTON DESINGED PCB 
+2.TO MAKE A ANDIOD APP WITH MULTIPLE LANGUAGE SUPPORT AN ALGORITHM TO LIST OUT PATIENTS WHO NEED MEDICAL ASSISTANCE ARGENTLY 
+3.PCB TOH CONNECT DIFFRENT MEDICAL PROBES WITH PI 
+4.TESTING WITH DIFFRENT MEDICAL SENSORS 
+5.DESINGING A METAL ENCLOSER TO PROTECT PI AND OTHER PCB .
+
+## Budget
+
+What materials will you need for your project? Where will you get them? How much does it cost? Please include all materials, including components you already own. Make sure to factor in shipping costs and sales tax.
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| SOLDRING KIT    | https://robu.in/product/plusivo-soldering-kit-with-multimeter-eu-plug/ | $21.27  |
+| raspberyy pi 4B+ 8GB| https://www.amazon.in/Raspberry-Pi-8GB-Desktop-Computer/dp/B08B9XS3B6?th=1  | $169.18 |
```

Comment:
do you need an SD card and power adapter? also, how do you plan on connecting to and programming your raspberry pi?

Reply:
I have both with me I m thinking of using SSH with LAN cable and with putty

Reply:
sounds good!